docs/pages/eas-update/estimate-bandwidth.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Estimate bandwidth usage
+description: Learn how to estimate bandwidth usage for EAS Update.
+---
+
+# Understanding update bandwidth usage
+
+EAS Update enables an app to update its own non-native pieces (such as JS, styling, and images) over-the-air. This guide explains how bandwidth is consumed and how consumption can be optimized.
+
+## Bandwidth calculation breakdown
+
+Each subscription plan includes a predefined bandwidth allocation per month. Additionally, extra MAUs purchased through usage-based pricing receives an additional 40 MiB of bandwidth per MAU. This bandwidth determines the number of updates your users can download. Here's how to estimate bandwidth usage per update:
+
+- **Update size:** The key factor in bandwidth consumption is the size of update assets that are not already on the device. If an update only modifies the JavaScript portion of your app, users will only download the new Javascript. To begin an example, let's say the uncompressed Javascript portion generated during export is **10 MB**. Compression will further reduce its size.
+- **Compression ratio:** The level of compression depends on the file type. JavaScript and Hermes bytecode (commonly used in React Native apps) can be compressed, while images and icons are not automatically compressed. In the example above, a Hermes bytecode bundle achieves an estimated **2.6× compression ratio**, reducing the actual download size to:
+
+  ```text
+  10 MB / 2.6 ≈ 3.85 MB update bandwidth size
+  ```
+
+- **Total update bandwidth:** Given a bandwidth allocation, we estimate how many updates can be downloaded. For example, if you have 60,000 MAUs on a production plan, it includes 50,0000 MAU and **1 TiB (1,024 GiB) of bandwidth per month**. An additional 10,000 MAUs purchased through usage-based pricing receives an additional **40 MiB of bandwidth per MAU**. The total number of updates that can be downloaded is:
```

Comment:
The first two bullets make sense under "Here's how to estimate bandwidth usage per update", but the third bullet seems like less of "how" and more of an example of applying the "how", so I would consider breaking it out of the bullet points.